Changelog.md

```diff
@@ -1,13 +1,18 @@
 Changelog
 =========
 
-### 2.0.0-beta2 (xxxx-xx-xx)
-
-* Use ceil in `ResettingController` for a better token lifetime approximation
-* Removed unused translation keys
-* Removed form deprecations
-* Added `symfony/templating` as a dependency
-* Use `@`-based Twig syntax for templates
+### 2.0.0-beta2 (2017-01-31)
+
+* Use ceil in `ResettingController` for a better token lifetime approximation.
+* Removed unused translation keys.
+* Removed form deprecations.
+* Added `symfony/templating` as a dependency.
```

Comment:
As said before, I would remove this from the changelog. We have not added a new dependency (it was there through FrameworkBundle).
And the switch to the native syntax actually removed the need for this dependency AFAIK